Codigo/Declares.bas

```diff
@@ -969,7 +971,7 @@ Public Type t_UserOBJ
     ObjIndex As Integer
     amount As Integer
     Equipped As Byte
-
+    LastUseTime As Long
```

Comment:
Rename to something better, `LastUseTime` does not mention what is to be used...

Reply:
its the last use time of the object, its inside a t_UserObj so it refers to the last time this object was used